Makefile

```diff
@@ -46,6 +46,7 @@ includes = \
 	board/$(BOARD) \
 	include \
 	include/platform \
+	include/sched-rr \
```

Comment:
Don't name it after `sched-rr`. Instead, use `include/sched` and manipulate each scheduling policy through dedicated file.